src/main/scala/no/ndla/audioapi/service/ConverterService.scala

```diff
@@ -8,51 +8,53 @@
 
 package no.ndla.audioapi.service
 
-import com.netaporter.uri.Uri
 import com.typesafe.scalalogging.LazyLogging
-import com.netaporter.uri.dsl._
 import no.ndla.audioapi.AudioApiProperties._
 import no.ndla.audioapi.auth.User
-import no.ndla.audioapi.model.api.NotFoundException
-import no.ndla.audioapi.model.domain.{Audio, Tag, Title}
-import no.ndla.audioapi.model.Language._
+import no.ndla.audioapi.model.Language.{findByLanguageOrBestEffort, DefaultLanguage}
 import no.ndla.audioapi.model.{api, domain}
 import no.ndla.mapping.License.getLicense
 
-import scala.util.{Failure, Success, Try}
+import scala.util.{Success, Try}
 
 
 trait ConverterService {
   this: User with Clock =>
   val converterService: ConverterService
 
   class ConverterService extends LazyLogging {
-    def toApiAudioMetaInformation(audioMeta: domain.AudioMetaInformation, language: String): Try[api.AudioMetaInformation] = {
-      val lang = language match {
-        case AllLanguages if audioMeta.supportedLanguages.contains(DefaultLanguage) => DefaultLanguage
-        case AllLanguages if audioMeta.supportedLanguages.nonEmpty => audioMeta.supportedLanguages.head
-        case l => l
-      }
-
-      if (!audioMeta.supportedLanguages.contains(lang))
-        return Failure(new NotFoundException)
 
-      val audioFile = findByLanguage(audioMeta.filePaths, lang).getOrElse(audioMeta.filePaths.head)
+    def toApiAudioMetaInformation(audioMeta: domain.AudioMetaInformation, language: Option[String]): Try[api.AudioMetaInformation] = {
       Success(api.AudioMetaInformation(
         audioMeta.id.get,
         audioMeta.revision.get,
-        lang,
-        findByLanguage(audioMeta.titles, lang).getOrElse(""),
-        toApiAudio(audioFile),
+        toApiTitle(findByLanguageOrBestEffort[String, domain.Title](audioMeta.titles, language)),
```

Comment:
Egentlig et spørsmål. 
Denne typeinformasjonen i kallet (`[String, domain.Title]`) i dette tilfellet) skjønte jeg ikke umiddelbart. Er den nødvendig? (eller kan det gjøres enklere)
Og hva er "betydningen" på de to verdiene du sender inn?